qdrant-landing/content/articles/multitenancy.md

```diff
@@ -4,10 +4,9 @@ short_description: "Explore how Qdrant's multitenancy and custom sharding stream
 description: "Discover how multitenancy and custom sharding in Qdrant can streamline your machine-learning operations. Learn how to scale efficiently and manage data securely."
 social_preview_image: /articles_data/multitenancy/social_preview.png
 preview_dir: /articles_data/multitenancy/preview
-small_preview_image: /articles_data/multitenancy/icon.svg
 weight: -120
 author: David Myriel
-date: 2024-02-06T13:21:00.000Z
+date: 2025-30-06T13:21:00.000Z
```

Comment:
@davidmyriel  I think we use YYYY-MM-DD date format, so this article won't be visible when you merge it ;)